pkg/apis/storage/v1/zz_generated.conversion.go

```diff
@@ -233,6 +253,16 @@ func Convert_storage_VolumeAttachmentList_To_v1_VolumeAttachmentList(in *storage
 
 func autoConvert_v1_VolumeAttachmentSource_To_storage_VolumeAttachmentSource(in *v1.VolumeAttachmentSource, out *storage.VolumeAttachmentSource, s conversion.Scope) error {
 	out.PersistentVolumeName = (*string)(unsafe.Pointer(in.PersistentVolumeName))
+	if in.InlineVolumeSpec != nil {
+		in, out := &in.InlineVolumeSpec, &out.InlineVolumeSpec
+		*out = new(core.PersistentVolumeSpec)
+		// TODO: Inefficient conversion - can we improve it?
```

Comment:
we shouldn't have reflect conversions here... this means the generator couldn't find the core conversions for inline volume spec... look at other places we use types from core (like podtemplatespec) in non-core types (like apps/Deployments) and see how they got the core conversion functions referenced correctly

Reply:
Thanks for the help with resolving this Jordan!